### PR TITLE
package.json test scripts and travis :)

### PR DESCRIPTION
**Pull request by zhuangya**
i ammend `scripts` in package.json for easier test workflow like `npm test`

also add travis yml and badges

also please don't forget enable this project in travis

### CODE REVIEW

File: test/sandbox/richard-hendriks.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
+{
```
Comment (zhuangya) [inline test/sandbox/richard-hendriks.json]: hmm, i think i create this file by running `npm run grunt`

should i remove it? @hacksalot

**Comment by hacksalot** [comment]
Great work, @zhuangya! We can definitely get this merged. Reviewing now, will update within 24.

File: test/sandbox/richard-hendriks.json (hunk 1 above)
Comment (hacksalot) [inline test/sandbox/richard-hendriks.json]: Yep, `/sandbox` is generated and can be .gitignored for that matter.

**Comment by hacksalot** [comment]
Looks good-- enabled in Travis and ready to merge once the hendriks resume is removed. Also can you add Node 0.10, 0.11, and 0.12 to the covered versions (unless there are issues with same). Thanks!

**Comment by zhuangya** [comment]
@hacksalot it's all ready and green

Merry Christmas and Happy New Year!

**Pull request merged**